lxd/device/nic_bridged.go

```diff
@@ -200,18 +200,20 @@ func (d *nicBridged) validateConfig(instConf instance.ConfigReader) error {
 	// Check there isn't another NIC with the any of the same addresses specified on the same cluster member.
 	// Can only validate this when the instance is supplied (and not doing profile validation).
 	if d.inst != nil && (d.config["ipv4.address"] != "" || d.config["ipv6.address"] != "") {
-		err := d.state.Cluster.InstanceList(nil, func(inst db.Instance, p api.Project, profiles []api.Profile) error {
+		filter := db.InstanceFilter{
+			Project: "",                // All projects.
```

Comment:
Shouldn't that be `Project: "default`` since you skip anything that's not the default project below.

Reply:
No, because that would filter out instances that are in a non-default project, that doesn't have networks enabled. The filter below that is after the effective network project has been derived.

Reply:
Ah right, makes sense.